### PR TITLE
JW-339 Clean up of vr_utils_temp.h

### PR DESCRIPTION
**Pull request by mmasztalerczuk**

### CODE REVIEW

**Comment by mmasztalerczuk** [comment]
Retest this please

**Comment by mmasztalerczuk** [comment]
Retest this please

File: include/netlink.h [hunk 1]
```diff
@@ -59,7 +59,6 @@ struct nlmsghdr {
 #define NLMSG_MIN_TYPE  0x10    /* Linux reserves below values for control
                                  * messages */
 
-
```
Comment (michal-clapinski) [inline include/netlink.h]: No useless changes.

File: include/netlink.h [hunk 2]
```diff
@@ -78,5 +77,20 @@ struct nlattr {
 #define NLA_ALIGNTO 4
 #define NLA_ALIGN(len)  (((len) + NLA_ALIGNTO - 1) & ~(NLA_ALIGNTO - 1))
 #define NLA_HDRLEN  ((int) NLA_ALIGN(sizeof(struct nlattr)))
+#define NLMSG_ALIGN(len) ( ((len)+NLMSG_ALIGNTO-1) & ~(NLMSG_ALIGNTO-1) )
```
Comment (michal-clapinski) [inline include/netlink.h]: Already defined.

File: include/netlink.h [hunk 3]
```diff
@@ -78,5 +77,20 @@ struct nlattr {
...
+#define NLMSG_HDRLEN     ((int) NLMSG_ALIGN(sizeof(struct nlmsghdr)))
```
Comment (michal-clapinski) [inline include/netlink.h]: Already defined.

File: include/netlink.h [hunk 4]
```diff
@@ -78,5 +77,20 @@ struct nlattr {
...
+#define GENL_HDRLEN     NLMSG_ALIGN(sizeof(struct genlmsghdr))
+#define GENL_ID_CTRL 0x10
+#define IFNAMSIZ 16
+#define IFNAMSIZ 16
+#define AF_BRIDGE 7
+#define AF_NETLINK 0
+#define NLMSG_DONE  0x3
```
Comment (michal-clapinski) [inline include/netlink.h]: Already defined.

File: include/netlink.h [hunk 5]
```diff
@@ -78,5 +77,20 @@ struct nlattr {
 #define NLA_ALIGNTO 4
 #define NLA_ALIGN(len)  (((len) + NLA_ALIGNTO - 1) & ~(NLA_ALIGNTO - 1))
 #define NLA_HDRLEN  ((int) NLA_ALIGN(sizeof(struct nlattr)))
+#define NLMSG_ALIGN(len) ( ((len)+NLMSG_ALIGNTO-1) & ~(NLMSG_ALIGNTO-1) )
+#define NLMSG_HDRLEN     ((int) NLMSG_ALIGN(sizeof(struct nlmsghdr)))
+#define GENL_HDRLEN     NLMSG_ALIGN(sizeof(struct genlmsghdr))
+#define GENL_ID_CTRL 0x10
+#define IFNAMSIZ 16
+#define IFNAMSIZ 16
```
Comment (michal-clapinski) [inline include/netlink.h]: Seriously? These two lines are the same.

**Comment by mmasztalerczuk** [comment]
Retest this please

**Comment by michal-clapinski** [comment]
Retest this please

File: include/netlink.h [hunk 6]
```diff
@@ -78,5 +78,16 @@ struct nlattr {
 #define NLA_ALIGNTO 4
 #define NLA_ALIGN(len)  (((len) + NLA_ALIGNTO - 1) & ~(NLA_ALIGNTO - 1))
 #define NLA_HDRLEN  ((int) NLA_ALIGN(sizeof(struct nlattr)))
+#define GENL_HDRLEN     NLMSG_ALIGN(sizeof(struct genlmsghdr))
+#define GENL_ID_CTRL 0x10
+#define IFNAMSIZ 16
```
Comment (michal-clapinski) [inline include/netlink.h]: Is this related to netlink?

File: include/netlink.h [hunk 7]
```diff
@@ -78,5 +78,16 @@ struct nlattr {
...
+#define AF_BRIDGE 7
+#define AF_NETLINK 0
```
Comment (michal-clapinski) [inline include/netlink.h]: Why weren't those two lines defined on FreeBSD? Do you know how does it work there?

File: include/netlink.h (hunk 6 above)
Comment (mmasztalerczuk) [inline include/netlink.h]: Yes

File: include/netlink.h (hunk 7 above)
Comment (mmasztalerczuk) [inline include/netlink.h]: I don't know why, but FreeBSD don't use ini_parser.c (where this defines is needed)

**Comment by mmasztalerczuk** [comment]
Retest this please
